internal/setup/yaml.go

```diff
@@ -161,7 +172,7 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 	if yamlVar.Format != "v1" {
 		return nil, fmt.Errorf("%s: unknown format %q", fileName, yamlVar.Format)
 	}
-	if len(yamlVar.Archives) == 0 {
+	if len(yamlVar.Archives)+len(yamlVar.V2Archives) == 0 {
```

Comment:
The temporal variable `len(yamlVar.Archives)+len(yamlVar.V2Archives)` is used twice, can we give it a name?

Reply:
I don't think it's that big of a deal to have a variable for this. Let me know if you have strong preferences for this.